draft.install

```diff
@@ -1,19 +1,42 @@
 <?php
 
 use Drupal\Core\Config\FileStorage;
+use Drupal\user\Entity\Role;
 use Drupal\user\Entity\User;
 
 /**
- * Implements hook_install().
+ * Implements hook_install_tasks().
+ *
+ * Performs tasks to set up the site for the Draft profile.
+ *
+ * @see install_tasks()
+ */
+function draft_install_tasks(&$install_state) {
+  return [
+    'draft_set_role' => [],
```

Comment:
Sort them in code to match the order in the hook

Reply:
They have been sorted